### PR TITLE
Add html css popup

### PR DESCRIPTION
**Pull request by VBystrov**
Hello!
The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by krylenger** [comment]
it's better to add the direct link to your Demo and code base. 
check out other PR's for an example

**Comment by VBystrov** [comment]
Here it is:
[demo](https://vbystrov.github.io/html-css-popup/index.html)
[code](https://github.com/VBystrov/html-css-popup)

**Comment by krylenger** [comment]
Well done! 
A couple of things to improve: 

this is a part of the task: 
_-You'll must able to switch across all interactive elements using only keyboard(without a mouse, don't forget about styles for :focus state)_

It's a good idea to implement this and improve you skill in accessibility.

File: submissions/VBystrov/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <title>Popup</title>
+    <meta name="description" content="HTML & CSS practice: Hooli-style Popup" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+
+  <body>
+    <header>
+      <div class="menu">
+        <a class="link" href="#">Mail</a>
```
Comment (krylenger) [inline submissions/VBystrov/html-css-popup/index.html:14]: some unofficial header navigation structure: 
header > ul > li > a

File: submissions/VBystrov/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        <a class="link" href="#">Images</a>
```
Comment (madmaxWMFU) [inline submissions/VBystrov/html-css-popup/index.html:15]: Navigation is best used in this format
```
<nav>
  <ul>
    <li><a href="#">Home</a></li>
    <li><a href="#">About</a></li>
    <li><a href="#">Contact</a></li>
  </ul>
</nav>
```

File: submissions/VBystrov/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <label
+          for="popup-checkbox"
+          class="popup-button link"
+          href="#popup-window"
+        ></label>
+        <a class="profile link" href="#"></a>
+      </div>
+      <input type="checkbox" id="popup-checkbox" />
+      <div class="popup-window" id="popup-window">
```
Comment (madmaxWMFU) [inline submissions/VBystrov/html-css-popup/index.html:24]: The Popup-menu is part of the main menu.

File: submissions/VBystrov/html-css-popup/index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+
+  <body>
+    <header>
+      <div class="menu">
+        <a class="link" href="#">Mail</a>
+        <a class="link" href="#">Images</a>
+        <label
```
Comment (madmaxWMFU) [inline submissions/VBystrov/html-css-popup/index.html:16]: This element hasn't focus

File: submissions/VBystrov/html-css-popup/index.html [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+          <a class="popup-item horn" href="#"></a>
+          <a class="popup-item microphone" href="#"></a>
+          <a class="popup-item metronome" href="#"></a>
+          <a class="popup-item purse" href="#"></a>
+          <a class="popup-item rock" href="#"></a>
+          <a class="popup-item jossstick" href="#"></a>
+        </div>
+
+        <input type="checkbox" id="popup-more" />
+        <label class="popup-more-label" for="popup-more"> MOAR</label>
```
Comment (madmaxWMFU) [inline submissions/VBystrov/html-css-popup/index.html:41]: This element hasn't focus also.

File: submissions/VBystrov/html-css-popup/style.css [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+  display: grid;
+  padding: 20px;
+  grid-template-columns: repeat(3, 1fr);
+  grid-auto-rows: 100px;
+  grid-gap: 10px;
+  justify-content: start;
+  align-content: start;
+}
+
+#popup-checkbox,
```
Comment (madmaxWMFU) [inline submissions/VBystrov/html-css-popup/style.css:61]: Avoid styling by id, better to use the class name

File: submissions/VBystrov/html-css-popup/index.html (hunk 4 above)
Comment (madmaxWMFU) [inline submissions/VBystrov/html-css-popup/index.html:16]: Attribute `href` not allowed on the element `label` at this point.

File: submissions/VBystrov/html-css-popup/index.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
```
Comment (madmaxWMFU) [inline submissions/VBystrov/html-css-popup/index.html:2]: Consider adding a `lang` attribute to the `html` start tag to declare the language of this document.

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Good job!
See some comments below

**Review by madmaxWMFU — APPROVED** [review]

**Pull request merged**